projects/plugins/jetpack/extensions/blocks/story/story.php

```diff
@@ -182,6 +182,30 @@ function get_image_crop_class( $width, $height ) {
 	return $crop_class;
 }
 
+/**
+ * Returns a URL for the site icon.
+ *
+ * @param int    $size - Size for (square) sitei icon.
+ * @param string $fallback - Fallback URL to use if no site icon is found.
+ *
+ * @returns string
+ */
+function get_blavatar_or_site_icon_url( $size, $fallback ) {
```

Comment:
Would it be worth pulling this from the `Jetpack_PostImages` class instead?
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/class.jetpack-post-images.php#L472-L477

Reply:
Sure! I refactored it to rely on `Jetpack_PostImages` in acbcc8 - let me know if you had a different approach than that in mind.